src/main/scala/org/broadinstitute/hail/stats/FisherExactTest.scala

```diff
@@ -0,0 +1,270 @@
+package org.broadinstitute.hail.stats
+
+import org.apache.commons.math3.distribution.HypergeometricDistribution
+import org.broadinstitute.hail.Utils.{fatal, divOption}
+
+object FisherUtils {
+  //based on C code in R source code called zeroin.c
```

Comment:
Can you provide a link to the version you based this on?  Also, I prefer a space after the `//` in comments.